packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -274,8 +270,15 @@ gmaps.Icon? _gmIconFromBitmapDescriptor(BitmapDescriptor bitmapDescriptor) {
     // Grab the bytes, and put them into a blob
     final List<int> bytes = iconConfig[1]! as List<int>;
     // Create a Blob from bytes, but let the browser figure out the encoding
-    final Blob blob = Blob(<dynamic>[bytes]);
-    icon = gmaps.Icon()..url = Url.createObjectUrlFromBlob(blob);
+    final Blob blob;
+
+    if (bytes is Uint8List) {
+      blob = Blob(<JSUint8Array>[bytes.toJS].toJS);
+    } else {
+      blob = Blob(<JSUint8Array>[Uint8List.fromList(bytes).toJS].toJS);
```

Comment:
Have you found a case where `bytes` is *not* a `Uint8List`? (is the else ever going to run?)

If this is just to cover the "else", just in case, I'd prefer to do:

```dart
assert(bytes is Uint8List, 'bytes must be an Uint8List (or something helpful)');
// Create the blob
```

And then see if anybody trips in that assertion :)